packages/snaps-cli/package.json

```diff
@@ -65,7 +66,6 @@
     "browserify-zlib": "^0.2.0",
     "buffer": "^6.0.3",
     "chalk": "^4.1.2",
-    "chokidar": "^3.5.2",
```

Comment:
Do we not use this anymore?

Reply:
Not since the Webpack rewrite. We use Webpack's watcher (likely still Chokidar, but no need to depend on it directly).